plugins/woocommerce/client/blocks/assets/js/blocks/payment-method-icons/block.json

```
@@ -0,0 +1,27 @@
+{
+	"name": "woocommerce/payment-method-icons",
+	"version": "1.0.0",
+	"title": "Payment Method Icons",
+	"description": "Display icons for available payment methods.",
```

If it only lists icons available via WooPayments, we should probably clarify it:

```suggestion
	"description": "Display icons for available WooPayments methods.",
```

---

That makes sense. I'd like for this block to handle multiple payment gateways eventually, but at the moment for simplicity, it only handles WooPayments. Updated the description in https://github.com/woocommerce/woocommerce/pull/57181/commits/ab437e3dfa1eb513b086453fc43c22067872c467.